internal/worker/task.go

```diff
@@ -72,16 +74,21 @@ func (worker *Worker) runTask(
 		}
 
 		// Provide tags for Sentry: task ID and upstream worker name
-		cirrusSentryTags := []string{
-			fmt.Sprintf("cirrus.task_id=%d", agentAwareTask.TaskId),
-			fmt.Sprintf("cirrus.upstream_worker_name=%s", upstream.WorkerName()),
+		cirrusSentryTags := map[string]string{
+			"cirrus.task_id":              strconv.FormatInt(agentAwareTask.TaskId, 10),
+			"cirrus.upstream_worker_name": upstream.WorkerName(),
 		}
```

Comment:
What do you think about calling:

```go
sentry.ConfigureScope(func(scope *sentry.Scope) {
	scope.SetTags(cirrusSentryTags)
})
```

...here, to avoid repeating `scope.SetTags(cirrusSentryTags)` multiple times below?

Reply:
We'll need to not forget pop the scope. That's why I opted into just doing everything at once.

Reply:
Plus I think it might conflict when we run multiple tasks in parallel